packages/Chem/src/descriptors/const.ts

```diff
@@ -0,0 +1,1321 @@
+export const descriptors: any = {
```

Comment:
Maybe, it's better to store it in JSON file, and not to build in to the package?